src/api/assetApi.ts

```diff
@@ -16,6 +18,10 @@ export class RestAssetApi extends RestApi implements AssetApi {
   fetchAvailableAssets = () => this.get('/assets/available');
   fetchAssetInstruments = () => this.get('/assetpairs');
   fetchRates = () => this.get('/markets');
+  fetchAssetAddress = (id: string) =>
+    this.get(`/deposits/crypto/${id}/address`);
+  generateAssetAddress = (id: string) =>
+    this.post(`/deposits/crypto/${id}/address`, undefined);
```

Comment:
Undefined as a parameter always looks very strange. Can it be something from function parameters?